app/salesGPT/components/message.tsx

```diff
@@ -1,10 +1,13 @@
 import MessageActions from "./messageActions";
 import styles from "./message.module.scss";
 
-function Message(props: { message: string }) {
+function Message(props: { title?: string; message: string }) {
   return (
     <div className={styles["chat-message-container"]}>
-      <MessageActions message={props.message} />
+      <div className={styles["chat-message-header"]}>
+        {props.title && <span>{props.title}</span>}
```

Comment:
Kunne denne vært enda mer tydeligere at den er en overskrift med å bruke for eksempel h3?

Reply:
Vi burde nok ta en skikkelig gjennomgang på alt sånt mtp UU på et tidspunkt, men i chaten så er det en div så for å få det likt så er det akkurat nå lettest og bare ha det sånn.. Dessverre, det er veldig dårlig praksis